src/python/gudhi/cubical/_cubical_complex.cc

```diff
@@ -0,0 +1,91 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2020 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#include <gudhi/Bitmap_cubical_complex.h>
+#include <gudhi/Bitmap_cubical_complex_base.h>
+#include <gudhi/Bitmap_cubical_complex_periodic_boundary_conditions_base.h>
+
+#include <Persistent_cohomology_interface.h>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <memory>       // for std::unique_ptr
+
+namespace py = pybind11;
+using namespace Gudhi::cubical_complex;
+
+
+template<typename CubicalComplexOptions = Bitmap_cubical_complex_base<double>>
+class Cubical_complex_interface_ : public Bitmap_cubical_complex<CubicalComplexOptions> {
```

Comment:
Is this class useful, or could we use directly Bitmap_cubical_complex?

Reply:
Remove useless class on 81c4df53